roc-wrapper.go

```diff
@@ -0,0 +1,15 @@
+package roc
+
+import "fmt"
+
+func convertErr(err uint32, errstr string) error {
+	if err == 0 {
+		return nil
+	}
+	return fmt.Errorf(errstr)
+}
+
+// Init parses the `ip` and `port` and initializes the Address object
+func (a *Address) Init(family Family, ip string, port uint32) error {
```

Comment:
I suggest to replace this with:

```
func NewAddress(family Family, ip string, port uint32) (*Address, error)
```

It would be more go-way. I also suggest to use the same approach for other objects, e.g.:

```
func OpenSender(...) (*Sender, error)
func (sender *Sender) Bind(...) error
...
```

Also, IIRC, Go stdlib usually use `int` for ports, so we can follow the convention and replace `port uint32` with `port int` here.

Reply:
done! check out my most recent commits!